app/helpers/nav_helper.rb

```diff
@@ -54,6 +54,10 @@ def show_sub_nav?(school, hide_subnav)
     school.present? && school.id && hide_subnav.nil?
   end
 
+  def header_fix_enabled?
+    !!@header_fix_enabled
```

Comment:
would `@header_fix_enabled == true` work here?

Reply:
Yes I think it would, thank you. (My thinking was that @header_fix_enabled might not be defined so doing it this way might cause an error. Just tested this theory though and this would be only the case for local variables and not instance variables. So I'll change it, thank you :-) )